src/status_im/ui2/screens/common/core.cljs

```diff
@@ -0,0 +1,4 @@
+(ns status-im.ui2.screens.common.core
+  (:require status-im.ui2.screens.common.alert.view))
+
+(def alert status-im.ui2.screens.common.alert.view/alert)
```

Comment:
could you elaborate?

Reply:
This is a file to allow importing common components from the common namespace without having to import each component (like in quo2)